src/super_gradients/training/metrics/detection_metrics.py

```diff
@@ -35,6 +38,10 @@ class DetectionMetrics(Metric):
                                             If True, the compute() function will return a metrics dictionary that not
                                             only includes the average metrics calculated across all classes,
                                             but also the optimal score threshold overall and for each individual class.
+    :param include_classwise_ap:            Whether to include the class-wise average precision in the returned metrics dictionary.
```

Comment:
Canyou please add a simple example to show what the output dict would look like ?

Reply:
Fixed in 3e45cdee841fc29df877611cd9eece46f84d0f6b